### PR TITLE
RN CD July 14

### PR DESCRIPTION
**Pull request by michaelcretzman**
RN PR: https://64b05bda7d9564440ef1aa97--harness-developer.netlify.app/release-notes/continuous-delivery

### CODE REVIEW

File: release-notes/continuous-delivery.md [hunk 1]
```diff
@@ -17,34 +17,150 @@ Harness publishes security advisories for every release. Go to the [Harness Trus
...
+  Specifying an image by digest, rather than just tag, is useful when you want to ensure that the image you deploy for a service is fixed and immutable. If an image with the specified tag/digest combination does not exist in the artifact registry, the pipeline execution fails.
   
 ```mdx-code-block
   </TabItem>
   <TabItem value="Fixed issues">
 ```
+
+- The `<+configFile.getAsBase64()>` expression not resolving correctly when the content had new lines. (CDS-73424)
+  
+  The issue occurred with new line characters while encoding config files. This is fixed and Harness now replaces new line characters with unicode.
```
Comment (douglas-j-bothwell) [inline release-notes/continuous-delivery.md:73]:
```suggestion
  The issue occurred with newline characters while encoding config files. This is fixed and Harness now replaces newline characters with unicode.
```

File: release-notes/continuous-delivery.md [hunk 2]
```diff
@@ -17,34 +17,150 @@ Harness publishes security advisories for every release. Go to the [Harness Trus
...
+- The sort order on the pipelines list page was incorrect. (CDS-73216)
+  
+  Now, Harness supports case-insensitive sorting for pipelines, input sets, and pipeline executions.
+- Artifactory artifact source **Artifact Name** regex value was not working correctly. (CDS-73150)
+  
+  Harness has added support for regex values for generic type Artifactory artifacts.
+- Cron triggers artifact setting fails when modified regex does not match any build. (CDS-72589, ZD-46323)
```
Comment (douglas-j-bothwell) [inline release-notes/continuous-delivery.md:80]:
```suggestion
- Cron triggers artifact setting failed when modified regex did not match any build. (CDS-72589, ZD-46323)
```

File: release-notes/continuous-delivery.md [hunk 3]
```diff
@@ -17,34 +17,150 @@ Harness publishes security advisories for every release. Go to the [Harness Trus
...
+  
+  Harness initially modifies the regex to see if it matches any build. The trigger was failing if it did not match. Now, if the regex does not match any build Harness will use the original regex.
```
Comment (douglas-j-bothwell) [inline release-notes/continuous-delivery.md:82]:
```suggestion
  Harness initially modifies the regex to see if it matches any build. The trigger was failing if it did not match. Now, if the regex does not match any build, Harness will use the original regex.
```

File: release-notes/continuous-delivery.md [hunk 4]
```diff
@@ -17,34 +17,150 @@ Harness publishes security advisories for every release. Go to the [Harness Trus
...
+- Deserialization of `isGitDefaultBranch` in `ScmGitMetadata` is failing in platform service. (CDS-73805)
```
Comment (douglas-j-bothwell) [inline release-notes/continuous-delivery.md:83]:
```suggestion
- Deserialization of `isGitDefaultBranch` in `ScmGitMetadata` was failing in platform service. (CDS-73805)
```

File: release-notes/continuous-delivery.md [hunk 5]
```diff
@@ -17,34 +17,150 @@ Harness publishes security advisories for every release. Go to the [Harness Trus
...
+  
+  There was an issue when deserializing the `isGitDefaultBranch` value when fetching Git metadata. This has now been fixed.
+- Error while trying to access a stage template. (CDS-73138, ZD-46636)
+  
+  Opening the template inputs UI drawer inside the template studio was breaking the page. This happened only for the specific template use case involving setup groups. This is now resolved.
+- Red and green colors implemented in some areas need to be consistent to avoid contrast issues. (CDS-73054)
+  
+  Improved the UI on the **Account Overview** page to distinguish the red and green metrics on the dashboard.
+- Background step with matrix is failing to find the entry point.	(CDS-73034, ZD-46534)
```
Comment (douglas-j-bothwell) [inline release-notes/continuous-delivery.md:92]:
```suggestion
- Background step with matrix was failing to find the entry point.	(CDS-73034, ZD-46534)
```

File: release-notes/continuous-delivery.md [hunk 6]
```diff
@@ -17,34 +17,150 @@ Harness publishes security advisories for every release. Go to the [Harness Trus
...
+  
+  A user was unable to use matrix expressions when the object list in the matrix exactly matched the combination of background tasks. This issue has now been fixed.
+- Template YAML is not changing after changes in the UI. (CDS-72942, ZD-46501)	
+  
+  There was an issue with editing and saving account level templates in the YAML builder. This issue has now been fixed.
```
Comment (douglas-j-bothwell) [inline release-notes/continuous-delivery.md:97]:
```suggestion
  There was an issue with editing and saving account-level templates in the YAML builder. This issue has now been fixed.
```

File: release-notes/continuous-delivery.md [hunk 7]
```diff
@@ -17,34 +17,150 @@ Harness publishes security advisories for every release. Go to the [Harness Trus
...
+- Chained pipelines runtime error	with Barrier step. (CDS-72063, ZD-45997)
+  
+  When using the pipeline chaining feature, there was an runtime error when the chained pipeline had a Barrier step. This issue has now been fixed.
+- The Harness Approvals auto-reject filter was only using **Created At** field. (CDS-72058, ZD-45810)
+  
+  The auto-reject feature in Harness Approval steps did not consider the execution time.
+  
+  Executions of the pipelines waiting on Harness Approval steps were rejected without considering the time modality of the executions.
+  
+  For example, let's say there are 3 executions of the same pipeline (A, B, C, in the order in which their respective approval step started) These executions are waiting on a Harness Approval step with auto-reject enabled. When approving the execution B, only execution A was expected to get rejected, and not C. The issue is that C is also being getting rejected.
```
Comment (douglas-j-bothwell) [inline release-notes/continuous-delivery.md:113]:
```suggestion
  For example, let's say there are 3 executions of the same pipeline (A, B, C, in the order in which their respective approval step started). These executions are waiting on a Harness Approval step with auto-reject enabled. When approving the execution B, only execution A was expected to get rejected, and not C. The issue was that C was also getting rejected.
```

File: release-notes/continuous-delivery.md [hunk 8]
```diff
@@ -17,34 +17,150 @@ Harness publishes security advisories for every release. Go to the [Harness Trus
...
+  
+  This behavior has been fixed. Now, auto-reject only rejects previous executions waiting on a Harness Approval step. The start time of the Harness Approval step is used to decide which executions to reject.
+- Status and count mismatch in matrix wrapper. (CDS-72030)
+  
+  In the stage status count displayed for matrices in the pipeline execution details page, the Ignore Failed strategy will now be counted as a success. The count behavior will not change for Aborted status. Instead, it will continue to show in the failure count as before.
+- A Jenkins job URL containing a space is displayed with invalid formatting in the **Output** tab of Jenkins step execution. (CDS-71362, ZD-45614)
+  
+  For Jenkins step **Output**, we are replacing spaces in the Jenkins URL with `%20`.
+- The **Proceed With Default Values** information is showing on UI incorrectly.	(CDS-71168)
```
Comment (douglas-j-bothwell) [inline release-notes/continuous-delivery.md:122]:
```suggestion
- The **Proceed With Default Values** information was showing on UI incorrectly.	(CDS-71168)
```

File: release-notes/continuous-delivery.md [hunk 9]
```diff
@@ -17,34 +17,150 @@ Harness publishes security advisories for every release. Go to the [Harness Trus
...
+  We have introduced a modification to disallow the configuration of **Proceed With Default Values** as the post-retry action.
+  
+  The **Proceed With Default Values** action is only applicable for input timeout errors. This action allows the pipeline execution to continue with default values when users are unable to provide execution-time input within the specified time limit.
+  
+  With this change, users will no longer be able to set **Proceed With Default Values** as the post-retry action, ensuring that the action is only valid for handling input timeout errors scenarios.
+- The **Services** dashboard information is missing or incorrect for the latest deployment. (CDS-70856, ZD-45388)
```
Comment (douglas-j-bothwell) [inline release-notes/continuous-delivery.md:128]:
```suggestion
- The **Services** dashboard information was missing or incorrect for the latest deployment. (CDS-70856, ZD-45388)
```

File: release-notes/continuous-delivery.md [hunk 10]
```diff
@@ -17,34 +17,150 @@ Harness publishes security advisories for every release. Go to the [Harness Trus
...
+  
+  There was an issue in the **Services** dashboard when step group names had special characters in them. The issue led to incorrectly displaying names. This issue has now been fixed by sanitizing the names.
+- Rerun of  triggered pipeline throws error. (CDS-69387)
```
Comment (douglas-j-bothwell) [inline release-notes/continuous-delivery.md:131]:
```suggestion
- Rerun of triggered pipeline threw an error. (CDS-69387)
```

**Review by douglas-j-bothwell — APPROVED** [review]

File: release-notes/continuous-delivery.md [hunk 11]
```diff
@@ -17,34 +17,150 @@ Harness publishes security advisories for every release. Go to the [Harness Trus
...
   
 ```mdx-code-block
   </TabItem>
   <TabItem value="Fixed issues">
 ```
+
+- The `<+configFile.getAsBase64()>` expression not resolving correctly when the content had new lines. (CDS-73424)
+  
+  The issue occurred with new line characters while encoding config files. This is fixed and Harness now replaces new line characters with unicode.
+- The sort order on the pipelines list page was incorrect. (CDS-73216)
```
Comment (brian-f-harness) [inline release-notes/continuous-delivery.md:74]:
```suggestion
   This item requires Harness Delegate version 79900. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).

- The sort order on the pipelines list page was incorrect. (CDS-73216)
```

File: release-notes/continuous-delivery.md [hunk 12]
```diff
@@ -17,34 +17,150 @@ Harness publishes security advisories for every release. Go to the [Harness Trus
...
+  
+  Now, Harness supports case-insensitive sorting for pipelines, input sets, and pipeline executions.
```
Comment (brian-f-harness) [inline release-notes/continuous-delivery.md:76]:
```suggestion
  Now, Harness supports case-insensitive sorting for pipelines, input sets, and pipeline executions.
  This item requires Harness Delegate version 79xxx. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```

File: release-notes/continuous-delivery.md [hunk 13]
```diff
@@ -17,34 +17,150 @@ Harness publishes security advisories for every release. Go to the [Harness Trus
...
+- Artifactory artifact source **Artifact Name** regex value was not working correctly. (CDS-73150)
+  
+  Harness has added support for regex values for generic type Artifactory artifacts.
```
Comment (brian-f-harness) [inline release-notes/continuous-delivery.md:79]:
```suggestion
  Harness has added support for regex values for generic type Artifactory artifacts.
  This item requires Harness Delegate version 79xxx. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```

File: release-notes/continuous-delivery.md (hunk 4 above)
Comment (brian-f-harness) [inline release-notes/continuous-delivery.md:83]:
```suggestion
   This item requires Harness Delegate version 79xxx. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).

- Deserialization of `isGitDefaultBranch` in `ScmGitMetadata` is failing in platform service. (CDS-73805)
```

**Comment by brian-f-harness** [comment]
@michaelcretzman I added suggestions to add the following to delegate items:

This item requires Harness Delegate version 79xxx. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).

Also, should https://harness.atlassian.net/browse/CDS-73127 be included? Release Notes Candidate is set to Yes, but there isn't a Public Release Notes Summary.

**Review by roshnisarangadharan — APPROVED** [review]

File: release-notes/continuous-delivery.md [hunk 14]
```diff
@@ -17,34 +17,168 @@ Harness publishes security advisories for every release. Go to the [Harness Trus
 Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features and fixes that these release notes describe might not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page in your Harness account. 
 :::
 
-## Latest - July 06, 2023, version 79811
+## Latest - July 14, 2023, version 79909
```
Comment (roshnisarangadharan) [inline release-notes/continuous-delivery.md:20]:
```suggestion
## Latest - July 18, 2023, version 79916
```

File: release-notes/whats-new.md [hunk 15]
```diff
@@ -20,7 +20,35 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
 :::
 
 
-## Latest - July 07, 2023
+## Latest - July 14, 2023
```
Comment (roshnisarangadharan) [inline release-notes/whats-new.md:23]:
```suggestion
## Latest - July 18, 2023
```

File: release-notes/whats-new.md [hunk 16]
```diff
@@ -20,7 +20,35 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+
+### Continuous Delivery
```
Comment (roshnisarangadharan) [inline release-notes/whats-new.md:25]:
```suggestion
### Continuous Delivery, version 79916
```

**Review by roshnisarangadharan — APPROVED** [review]

File: release-notes/early-access.md [hunk 17]
```diff
@@ -13,24 +13,40 @@ Additionally, Harness publishes security advisories for every release. Go to the
 Harness deploys changes to Harness SaaS clusters on a progressive basis. This means that the features described in these release notes may not be immediately available in your cluster. To identify the cluster that hosts your account, go to the **Account Overview** page.
 :::
 
-## Latest - July 12, 2023
+## July 18, 2023
```
Comment (roshnisarangadharan) [inline release-notes/early-access.md:16]:
```suggestion
## Latest - July 18, 2023
```

**Pull request merged**